crimsobot/bot.py

```diff
@@ -150,7 +150,9 @@ async def on_message(self, message: discord.Message) -> None:
         cleaner = commands.clean_content(use_nicknames=False)
 
         # respond to ping or randomly talk
-        if self.user in message.mentions or (random.random() < 0.002 and not is_dm):
+        if (message.author.id != self.user.id and
```

Comment:
This is potentially unsafe as it does not limit other bots from triggering markov - I suggest changing `message.author.id != self.user.id` to something that checks against `message.author.bot`, as that will catch both other bots and crimsoBOT itself.